examples/simple-scene/Cargo.toml

```diff
@@ -6,9 +6,10 @@ version = "0.2.0"
 
 [lib]
 crate-type = ["lib", "cdylib"]
+name = "hotham_simple_scene_example"
 
 [[bin]]
-name = "hotham_simple_scene_example"
+name = "hotham_simple_scene_example_desktop"
```

Comment:
The suggested name in the issue description #364 is `simple_scene`.

Reply:
fixed in [this commit](https://github.com/leetvr/hotham/pull/406/commits/2498d02b2cf33e8100afaac2901e18b5de2c932b)

Reply:
I think the `hotham_` prefix is the way to go! Sorry, I'll clarify that in the issue. 😄

Reply:
> I think the `hotham_` prefix is the way to go! Sorry, I'll clarify that in the issue. 😄

curious, what's the thinking behind the prefix? feels more ergonomic to me to have it as `simple_scene_example_desktop`